### PR TITLE
Regenerates yarn.lock

### PR DESCRIPTION
**Pull request by SiAshbery**
#### Trello Ticket / Github Issue
https://trello.com/c/vmhPOXki/246-review-dependencies

#### Description
Solves dependency vulnerabilities by regenerating yarn.lock and updating imagemin-cli to 4.0.0

5 vulnerabilities remain:
2x High
3x Low

#### Test 
Website should continue to function without change or error.

### CODE REVIEW

File: site/fetchers/blog-posts/index.js [hunk 1]
```diff
@@ -55,9 +55,9 @@ const getPostsForTag = params =>
 
 /* eslint-disable camelcase */
 const getPosts = (tags: Array<string>) =>
-  (Promise.all(tags.map(topicId => getPostsForTag({ topic_id: topicId }))).then(flatten): Promise<
-    any,
-  >);
+  (Promise.all(tags.map(topicId => getPostsForTag({ topic_id: topicId }))).then(
+    flatten,
+  ): Promise<any>);
```
Comment (dpiatek) [inline site/fetchers/blog-posts/index.js]: I wonder about these formatting changes - are we using different prettier configs (I'm sure I've touched these files recently)

File: site/layout/style.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import 'normalize.css/normalize.css';
-@import 'reset-css/reset.css';
+@import '~normalize.css/normalize.css';
+@import '~reset-css/reset.css';
```
[the remark this answers is not in the document]
Comment (SiAshbery) [reply site/layout/style.css]: `@import` was importing relative to its current dir, the `~` resolves the module import correctly.

File: flow-typed/webpack.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// @noflow
```
[the remark this answers is not in the document]
Comment (SiAshbery) [reply flow-typed/webpack.js]: It's to prevents `flowtype/no-types-missing-file-annotation` https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-no-types-missing-file-annotation

I  wasn't sure if a refactor was the correct course of action, and this preserved the existing code. I think the core issue is my lack of understanding about flow.

File: site/fetchers/blog-posts/index.js (hunk 1 above)
Comment (SiAshbery) [reply site/fetchers/blog-posts/index.js]: I think you may be right. My linter also seems to want me to make everything a `.ts` file.

File: flow-typed/webpack.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (dpiatek) [reply flow-typed/webpack.js]: Ideally we would add types where they are missing, as we go. Bar tests, the whole codebase could benefit from types. There could be issues with those though and if it's something flow does not support (well) I'm fine with adding an `any` type on it, as long it's propagated.

File: site/fetchers/blog-posts/index.js (hunk 1 above)
Comment (dpiatek) [reply site/fetchers/blog-posts/index.js]: Potentially you need to flow and prettier plugins? They should read the config files in the repo.

**Pull request merged**